scripts/build.ps1

```diff
@@ -56,6 +59,10 @@ param (
     [ValidateSet("x86", "x64", "arm", "arm64")]
     [string]$Arch = "x64",
 
+    [Parameter(Mandatory = $false)]
+    [ValidateSet("UWP", "windows", "linux")] # For future expansion
```

Comment:
Let's keep the case the same for all (lowercase).

Reply:
Also, below, if $Platform is empty, default it below to "windows" or "linux" based on the current platform. Then you can use that when building up $ArtifactsDir and $BuildDir.